lib/python/qmk/cli/new/keyboard.py

```diff
@@ -143,36 +143,41 @@ def augment_community_info(src, dest):
 
 def prompt_keyboard():
     prompt = """{fg_yellow}Name Your Keyboard Project{style_reset_all}
-
 For more infomation, see:
 https://docs.qmk.fm/#/hardware_keyboard_guidelines?id=naming-your-keyboardproject
 
-keyboard Name? """
+Keyboard Name? """
 
-    return question(prompt, validate=lambda x: not keyboard(x).exists())
+    kb_name = False
```

Comment:
I had lent on the `validate=` of `question(` to originally provide the looping. The reasoning being that it would add in boilerplate like below. Maybe its more a case for milc on how it handles the prompt on validation failure.

Reply:
Yeah, I can't say like this either, but in this case I think the lack of error messages is the bigger evil.

I'll check what can be done form `milc` end for a longer-term solution.

Reply:
what about hiding it behind another function in the shorter term
```python
def prompt_keyboard():
    prompt = """{fg_yellow}Name Your Keyboard Project{style_reset_all}

For more infomation, see:
https://docs.qmk.fm/#/hardware_keyboard_guidelines?id=naming-your-keyboardproject

keyboard Name? """

    errmsg = f'Keyboard {{fg_cyan}}{kb_name}{{fg_reset}} already exists! Please choose a different name:'

    return _question(prompt, reprompt=errmsg, validate=lambda x: not keyboard(x).exists())
```

Reply:
I couldn't settle on anything that I liked any better so I ~~stole~~ used this code snippet. ¯\_(ツ)_/¯